### PR TITLE
grafana: Add Hubble dashboard

### PR DESCRIPTION
**Pull request by gandro**
This adds the Grafana dashboard for Hubble to our Grafana example chart.
Because `kubectl apply` has a limitation of 256KB per ConfigMap, the
chart is modified to create one ConfigMap per dashboard.

The "Running Prometheus & Grafana" getting started guide is updated
with information on how to enable metrics for Hubble and Cilium Operator.

In addition, the images from the Hubble metrics documentation are
copied over into the "Examples" section.

### CODE REVIEW

**Review by nebril — APPROVED** [review]

File: examples/kubernetes/addons/prometheus/files/grafana-dashboards/hubble-dashboard.json [hunk 1]
```diff
@@ -0,0 +1,3232 @@
...
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    }
+  ],
+  "refresh": "5s",
```
Comment (aanm) [inline examples/kubernetes/addons/prometheus/files/grafana-dashboards/hubble-dashboard.json]: 30s instead of 5s

**Review by aanm — CHANGES_REQUESTED** [review]
needs docs changes to inform users that `__meta_kubernetes_service_annotation_prometheus_io_scrape` needs to be set in prometheus. Can be addressed as a follow up if you prefer

**Review by rolinh — APPROVED** [review]

**Comment by gandro** [comment]
> needs docs changes to inform users that `__meta_kubernetes_service_annotation_prometheus_io_scrape` needs to be set in prometheus. Can be addressed as a follow up if you prefer

I have added it to this PR. Plus some other minor cosmetic fixes (such as making the links clickable).

**Comment by gandro** [comment]
test-me-please

**Comment by gandro** [comment]
retest-4.19

**Comment by michi-covalent** [comment]
for my own reference, the updated metrics page: https://deploy-preview-12004--cilium-docs.netlify.app/configuration/metrics.html

**Review by michi-covalent — APPROVED** [review]
@gandro thank you for taking care of this!

**Comment by gandro** [comment]
retest-4.19

**Review by aanm — APPROVED** [review]

**Pull request merged**